<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WinQuest</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	border: none;
	background-color: transparent;
	background: none;
	padding: 0;
	margin: 0;
	color: #FFF;
}

#centralwidget  {
	background-color: #5158BB;
}

.QLineEdit {
	background: transparent;
	border-bottom: 2px solid #F26DF9;
}
.QPushButton {
	background-color: #F26DF9;
	padding: 2px 5px;
}

#createAccountSecondaryButton {
	border: 2px solid #F26DF9;
	background: none;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="connectUsername">
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="placeholderText">
         <string>Nom d'utilisateur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="connectPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Mot de passe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>Se connecter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="connectErrorsLabel">
        <property name="styleSheet">
         <string notr="true">color: #F3A917</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createAccountSecondaryButton">
        <property name="text">
         <string>Créer un compte</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>500</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="accountUsername">
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="placeholderText">
         <string>Nom d'utilisateur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="accountPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Mot de passe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="accountPasswordConfirmation">
        <property name="text">
         <string>Confirmez votre mot de passe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createAccountButton">
        <property name="text">
         <string>Créer un compte</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>500</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="createAccountErrorsLabel">
        <property name="styleSheet">
         <string notr="true">color: #F3A917</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
